src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkUpdateService.java

```diff
@@ -537,7 +538,9 @@ public BulkRequestPayload bulkPreAcceptLogic(BulkDetails bulkDetails, List<Submi
                         bulkDetails.getCaseData().getMultipleReference(),
                         createUpdatesBusSender,
                         String.valueOf(ethosCaseRefCollection.size()),
-                        null);
+                        MultiplesHelper.generateMarkUp("ccdGatewayBaseUrl",
```

Comment:
this is passing in ccdGatewayBaseUrl and not the actual endpoint. What you had before was ok with Value was ok but it had the full library path.

Reply:
this is still passing it in as a string as opposed to the actual variable. shouldn't this need changing?